### ISSUE TITLE
PCSpeaker beep wont work

### ISSUE THREAD

**Issue by 365Tito** [CosmosOS/Cosmos#853]
whenever i use the beep function, no sound is played.

**Comment by fanoI** [comment]
Probably it is implemented as an empty plug, it needs to be properly plugged to make hardware speaker "beep".

**Comment by 365Tito** [comment]
I tried debugging it on Bochs and VMWare, on Bochs it works, but instead of a single beep (beep was set to trigger when user entered "BEEP" in the console) it makes a sound prettymuch like dial-up modems used to. You know, constant beeping and all.

**Comment by KrzysztofSzewczyk** [comment]
Can be resolved by porting this
```
 //Play sound using built in speaker
 static void soundstart(uint32_t freq) {
        //Recalibrate PIT to desired freq
     outportb(0x43, 0xb6);
     outportb(0x42, (uint8_t) (1193180 / freq) );
     outportb(0x42, (uint8_t) ((1193180 / freq) >> 8));

        //Play the sound
     uint8_t tmp = inportb(0x61);
      if (tmp != (tmp | 3)) {
         outportb(0x61, tmp | 3);
     }
 }

 static void soundstop() {
     outportb(0x61, inb(0x61) & 0xFC);
 }
```
To C#, and fixing PIT #867

**Comment by Arawn-Davies** [comment]
I can take a look at this, it should be an easy thing to do with PIT already in HAL
Is there a specific note that Console uses as default beep tone?

**Comment by Arawn-Davies** [comment]
Currently WIP in PR #952

**Comment by jp2masa** [comment]
Fixed by #952.
